Resources/Prototypes/Traits/physical.yml

```diff
@@ -742,7 +746,7 @@
 - type: trait
   id: Redshirt
   category: Physical
-  points: 8
+  points: 10
```

Comment:
This not justified, 8 points is already a lot for simply not having a crit state.